web/src/components/product/ProductRegistrationPage.tsx

```diff
@@ -174,6 +181,81 @@ const HostnameAlert = () => {
   );
 };
 
+const Extensions = () => {
```

Comment:
Honestly, I think this component deserves to live in it's own file and, if possible, having its own unit test.